site/tests/app/controllers/HomePageControllerTester.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace tests\app\controllers;
+
+use app\libraries\response\RedirectResponse;
+use app\libraries\response\WebResponse;
+use app\libraries\response\JsonResponse;
+use app\models\User;
+use app\libraries\Core;
+use tests\BaseUnitTest;
+use app\controllers\HomePageController;
+
+class HomePageControllerTester extends BaseUnitTest {
+
+	protected $controller;
```

Comment:
This is not used? Rename it to true_controller? Or rename true_controller to just controller?